### PR TITLE
Solving Build Error

### PR DESCRIPTION
**Pull request by deepak140596**
## Description of what I changed

Checking for build error on Travis
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by f4ww4z** [comment]
@deepak140596 See AC-594.

File: openmrs-client/build.gradle [hunk 1]
```diff
@@ -22,6 +22,7 @@ def version = "2.6.1-beta"
 android {
     compileSdkVersion 28
     useLibrary 'org.apache.http.legacy'
+    buildToolsVersion "27.0.3"
```
Comment (deepak140596) [inline openmrs-client/build.gradle]: no need to add this line

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,15 @@
 sudo: required
 language: android
 jdk: oraclejdk8
+dist: trusty
```
Comment (deepak140596) [inline .travis.yml]: This line is necessary for successful build.

**Comment by deepak140596** [comment]
Will close this PR once AC-594 is merged.

**Pull request closed without merging**